data/presets/amenity/clinic/dialysis.json

```diff
@@ -0,0 +1,24 @@
+{
+    "icon": "fas-kidneys",
```

Comment:
unfortunately, this icon is a _pro_ icon of fontawesome, and not available in the open source version of the icon set. :unamused: